js/core/utils.js

```diff
@@ -34,18 +34,32 @@ define(
         // if args.isDefinition is true, then the element is a definition, not a 
         // reference to a definition.  Any @title or @lt will be replaced with
         // @data-lt to be consistent with Bikeshed / Shepherd.
+        //
+        // This method now *prefers* the data-lt attribute for the list of 
+        // titles.  That attribute is added by this method to dfn elements, so 
+        // subsequent calls to this method will return the data-lt based list.
+        //
+        // This method will publish a warning if a title is used on a definition
+        // instead of an @lt (as per specprod mailing list discussion).
         $.fn.getDfnTitles = function ( args ) {
             var titles = [];
             var theAttr = "";
             var titleString = ""; 
-            if (this.attr("title")) {
-                titleString = this.attr("title");
-                theAttr = "title";
+            if (( !args || (args && args.isDefinition != true) ) && this.attr("data-lt")) {
+                // only look for a data-lt when we are NOT being called to create a
+                // new definition
+                titleString = this.attr("data-lt");
+                theAttr = "data-lt";
             }
             else if (this.attr("lt")) {
                 titleString = this.attr("lt");
                 theAttr = "lt";
             }
+            else if (this.attr("title")) {
+                titleString = this.attr("title");
+                theAttr = "title";
+                respecEvents.pub("warn", "Using deprecated attribute @title to specify link title for '" + this.text() + "': Change to @lt");
```

Comment:
Hmm.  Let me put together a pull request with better documentation in there first so the URI can be specific.